packages/component-library/draft/Kaizen/Modal/Presets/ConfirmationModal.scss

```diff
@@ -13,6 +13,11 @@
   max-width: 600px;
 }
 
+.elmModal {
```

Comment:
why did you call `elmModal`? is there any other modal overlapping?

Reply:
Yes there is a `.modal` style and it does differ in style implementation a bit. `.elmModal` does compose `.modal` but it adds a few different styles required by the elm version of Modal. This is mainly due to elm Modal being built around the same time as React version. Styling wise they are identical but there are a few minor differences.